test_scripts/Smoke/Resumption/ATF_Resumption_LIMITED_IGNITION_OFF.lua

```diff
@@ -0,0 +1,126 @@
+--  Requirement summary:
+--  [HMILevel Resumption]: Send BC.OnResumeAudioSource to HMI for the app resumed to LIMITED
+--
+--  Description:
+--  Check that SDL resumes media App in LIMITED HMI level if app was disconnected in less then 30 sec 
+--  before IGNITION_OFF and registered in less then after IGNITION_ON
+--
+--  1. Used preconditions
+--  App is registered and activated on HMI
+
+--  2. Performed steps
+--  Perform iginition off
+--  Perform ignition on
+--
+--  Expected result:
+--  1. SDL save all App's data to app_info.dat and sends to HMI OnSDLClose
+--  2. App is registered, SDL sends OnAppRegistered with the same HMI appID as in last ignition cycle, then sets App to LIMITED HMI level
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app = nil -- will be initialized after application registration
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            default_app = self.applications[default_app_params.appName]
```

Comment:
@LitvinenkoIra Please avoid to create unused variables.
Variable `default_app` was initialized but is not used.